documentation/modules/rn-2.5.adoc

```diff
@@ -26,27 +26,34 @@ In this version of {project-short}, migration using OpenStack source providers g
 
 .Disabling FIPS
 
-EMS enforcement is disabled for migrations with VMware vSphere source providers to enable migrations from versions of vSphere that are supported by {project-short} but do not comply with the 2023 FIPS requirements.
+With VMware vSphere source providers, Enterprise Master Secret (EMS) enforcement is disabled for migrations to enable migrations from versions of vSphere that are supported by {project-short} but do not meet the 2023 FIPS requirements.
 
 .Integration of the create and update provider user interface
 
 The user interface of create and update providers now aligns with the look and feel of the {ocp} web console and displays up-to-date data.
 
 .Standalone UI
 
-The old UI of MTV 2.3 cannot be enabled by setting `feature_ui: true` in ForkliftController anymore.
+The old UI of {project-short} 2.3 cannot be enabled by setting `feature_ui: true` in ForkliftController anymore.
 
-.Errors logged in populator pods are improved
+.Support deployment on OpenShift 4.15
+
+A vulnerability in Node.js Package Manager (npm) IP package that allowed an attacker to arbitrarily run code and obtain sensitive information. The operator was fixed on OpenShift 4.15, enabling MTV 2.5.6 to deploy the `match` property in the UI.
```

Comment:
this text seems to mix the fix for [MTV-941](https://issues.redhat.com/browse/MTV-941) (CVE) and the fix for deployment on 4.15

Reply:
@ahadas 
What should I write for this technical change?

Reply:
@ahadas 
Text changed to remove technical details.